RtspServer/RtspServer.cs

```diff
@@ -262,6 +258,8 @@ internal static void ConfigureRtspServerSocket(Socket socket, out int interFrame
         //Handles the Restarting of streams which needs to be and disconnects clients which are inactive.
         internal Timer m_Maintainer;
 
+        int m_MaintainerThreadId = 0;
```

Comment:
If this is just for the server can use the LocalEndpoint rather then introduce a new Id here, I thought this was on the RtspClient. Sorry for any confusion.

Reply:
"Maintenance-" + m_ServerThread.Name ?